iron-collapse.html

```diff
@@ -239,6 +239,14 @@
       this.notifyResize();
     },
 
+    _isVisible: function() {
+      var rect = this.getBoundingClientRect();
```

Comment:
I don't think this would work, because `_isVisible` is called on the pristine element, before its `style[this.dimension]` is set to the target size. That means that `_isVisible` would return `false` for say a top-level `iron-collapse` component that is displayed but closed; we don't want to skip the animation in that case.

In other words, `visible` here means "is not `display: none`", which is different from the `opened` state.